tests/Countries/NepalTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Spatie\Holidays\Tests\Countries;
+
+use Carbon\CarbonImmutable;
+use Spatie\Holidays\Holidays;
+
+it('can calculate nepal holidays', function () {
+    CarbonImmutable::setTestNowAndTimezone('2024-01-01');
```

Comment:
```suggestion
    CarbonImmutable::setTestNow('2024-01-01');
```

Reply:
Could you also run the tests locally and verify the generated results in the snapshot?